dashboard/test.js

```diff
@@ -0,0 +1,64 @@
+/*****
```

Comment:
This file should not exist. The dashboard part should be included in the main index.js.